DCS-265 mock responses for probation areas and local delivery units

GurnankCheema opened the pull request:
New mock responses for two new endpoints: 
/probationAreas
/probationAreas/code/:code/localDeliveryUnits

GurnankCheema committed:
DCS-265 mock responses for probation areas and local delivery units

GurnankCheema committed:
DCs-265: addition of new probation area and associated ldu

andrewrlee commented on routes/delius.js:
```diff
@@ -102,4 +102,91 @@ router.get('/offenders/nomsNumber/:nomsNumber/allOffenderManagers', (req, res) =
+    ],
+  }
+  res.send(response)
+})
```
As we were saying the other day, a nice refactoring exercise here, can remove the switch and duplicate references to `content` by creating some objects representing probation areas. ping me if you want to chat about it

GurnankCheema replied:
refactored

GurnankCheema committed:
DCS-265 code refactor

Have moved the probabtion area data into a separate module. The probation areas and their respective ldus are now complete objects enclosed in an array

andrewrlee commented on routes/stubs/probationAreas.js:
```diff
@@ -0,0 +1,79 @@
+module.exports = [
+  {
+    area: {
+      content: { code: 'Lon', description: 'London' },
```
this could be simplified a bit: you don't need to represent `content` blocks.
You can optimise it to suit the most simple way of representing the data:

```js
{ 
 code:  "Lon",
 description:  "London",
 ldus: [
        { code: 'ham', description: 'Hampstead' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead0' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead1' },
        { code: 'wtl', description: 'Waterloo1' },
        { code: 'pic', description: 'Picadilly1' },
        { code: 'ham', description: 'Hampstead' },
        { code: 'wtl', description: 'Waterloo' },
        { code: 'pic', description: 'Picadilly' },
        { code: 'ham', description: 'Hampstead1' },
        { code: 'wtl', description: 'Waterloo2' },
        { code: 'pic', description: 'Picadilly1' },
      ],
}
```

GurnankCheema replied:
I did wonder about that but I tried to keep it in line with the image in the ticket

andrewrlee replied:
the returned response needs to be in that format - but the structure we store that data can be optimised for that

GurnankCheema committed:
DCS-265: stub data restructured and methods using stub data adapted

andrewrlee reviewed (approved):
LGTM

bell-pa reviewed (approved):
LGTM
